docs/docs/how-tos/graph-api.ipynb

```diff
@@ -1288,10 +1288,33 @@
   },
   {
    "cell_type": "markdown",
-   "id": "4eeb895c-adca-40ab-b289-93ee56e18661",
+   "id": "6d99d63c",
    "metadata": {},
    "source": [
-    "</details>"
+    "## Add Node Caching\n",
+    "\n",
+    "There are many use cases where you may wish for your node to have a custom cache policy, for example if you conducting an expensive operation (either in terms of time or cost) that you want to avoid repeating. LangGraph lets you add individualized caching policies to nodes in a graph.\n",
```

Comment:
```suggestion
    "Node caching is useful in cases where you want to avoid repeating operations, like when doing something expensive (either in terms of time or cost). LangGraph lets you add individualized caching policies to nodes in a graph.\n",
```
suggested language change - can take it or leave